data/articles/progress-bar.md

```diff
@@ -0,0 +1,395 @@
+---
+id: 11
+author: "kkpiotrowski"
+topics: ["hanami", "htmx", "sidekiq", "rom-rb", "redis", "sequel"]
+title: "Hanami and HTMX progress bar"
+excerpt: "<<Summary>>"
+videoId: null
+published: false
+publishedAt: 
+modifiedAt: 
+aliases: ['HMEP011']
+thumbnail:
+  full: https://via.placeholder.com/1920/600?text=HanamiMastery.com
+  big: https://via.placeholder.com/780/440?text=HanamiMastery.com
```

Comment:
Maybe This?
https://unsplash.com/photos/text-jf1EomjlQi0

FYI: Cover images are stored in: `images/episodes/ID`, and the versions are as shown in those placeholders here.